dataprofiler/labelers/labeler_utils.py

```diff
@@ -184,24 +188,26 @@ def evaluate_accuracy(
                 f1_report["macro avg"][metric] = np.nan
             else:
                 f1_report["macro avg"][metric] *= (
-                    float(len(label_names)) / num_labels_with_positive_support
+                    float(len(label_names or [])) / num_labels_with_positive_support
                 )
 
     if "macro avg" in f1_report:
-        f1 = f1_report["macro avg"]["f1-score"]  # this is micro for the report
+        f1: float = f1_report["macro avg"]["f1-score"]  # this is micro for the report
     else:
         # this is the only remaining option for the report
         f1 = f1_report["accuracy"]
 
     if verbose:
-        f1_report_str = f1_report_dict_to_str(f1_report, label_names)
+        f1_report_str = f1_report_dict_to_str(f1_report, label_names or [""])
```

Comment:
same here

Reply:
what about same idea from comment earlier... add a condition to check `label_names` value...